RELEASE_NOTES.md

```diff
@@ -0,0 +1,5 @@
+#### 1.0.1 May 27 2016 ####
+Removed non-generic IPublisher and ISubscriber interfaces.
```

Comment:
Hi Chris,

this is what the -jvm project had as requirements for reaching 1.0.0:
(You can see some of the discussion here:
https://github.com/reactive-streams/reactive-streams-jvm/issues/223)

1. Spec "done" (it's in the README.md)
2. Interfaces done and documented (this you already have)
3. TCK "done" (this needs to be *carefully* migrated to .NET)
4. Examples written (with documentation) (I'm not sure how much work or if
it makes sense to port the -jvm examples to .NET)

*Graduation demonstration: At least 4 implementations passing the TCK which
can also be used successfully together.

The last bit is not strictly required since there's a lot of prior art in
place already for the JVM (i.e. we know the spec, interfaces and TCK work)


I cannot understate the importance of the TCK, as late as 2 days ago we
found a bug in an implementation thanks to it.

Please don't hesitate to ask if you have any questions, I'll do my best to
answer.


On Mon, May 30, 2016 at 4:37 PM, Chris Constantin <notifications@github.com>
wrote:

> In RELEASE_NOTES.md
> <https://github.com/reactive-streams/reactive-streams-dotnet/pull/7#discussion_r65079841>
> :
>
> > @@ -0,0 +1,5 @@
> > +#### 1.0.1 May 27 2016 ####
> > +Removed non-generic IPublisher and ISubscriber interfaces.
>
> Hi Victor,
>
> How would you define feature parity with the jvm version? The API is at
> feature parity I believe.
>
> I'll make the changes to publish this as a beta release.
>
> Thanks
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/reactive-streams/reactive-streams-dotnet/pull/7/files/dadf35ab71e8e40ca3ca852a2252bab85a820ffa#r65079841>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAqd6mLq0lPuF-9O57wuj47i1UO3OVkks5qGvY1gaJpZM4IoorW>
> .
>



-- 
Cheers,
√